src/components/Sections/Topics/SubTopics.js

```diff
@@ -29,35 +29,28 @@ const getItemsRecursive = (rootID, getById, base) => {
   return resp;
 };
 
-const SubTopics = ({ namespace, rootID, t }) => {
-  const [open, setOpen]             = useState(false);
-  const [isSelected, setIsSelected] = useState(false);
-  const [query, setQuery]           = useState();
+const SubTopics = ({ namespace, rootID }) => {
+  const [open, setOpen]   = useState(false);
+  const [query, setQuery] = useState();
 
+  const { t }           = useTranslation();
   const getTagById      = useSelector(state => tags.getTagById(state.tags));
   const getPathTags     = useSelector(state => tags.getPathByID(state.tags));
   const roots           = useSelector(state => tags.getRoots(state.tags));
   const baseItems       = useSelector(state => selectors.getTree(state.filtersAside, namespace, FN_TOPICS_MULTI));
   const selectedFilters = useSelector(state => filters.getFilterByName(state.filters, namespace, FN_TOPICS_MULTI));
   const selected        = useMemo(() => selectedFilters?.values || [], [selectedFilters]);
 
-  console.log('SELECTED', selected);
-
   const root  = getTagById(rootID);
   const items = useMemo(() => getItemsRecursive(rootID, getTagById, baseItems) || [], [rootID, getTagById, baseItems]);
 
   useEffect(() => setQuery(null), []);
-
-  useEffect(() => {
```

Comment:
Why did you remove this effect?

Reply:
its not depend on third side so not need effect